x-pack/test/functional/apps/dashboard/reporting/screenshots.ts

```diff
@@ -122,14 +122,32 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         const reportData = await PageObjects.reporting.getRawPdfReportData(url);
         const reportFileName = 'dashboard_preserve_layout';
         const sessionReportPath = await writeSessionReport(reportFileName, reportData, 'png');
-        const percentSimilar = await checkIfPngsMatch(
+        const percentDiff = await checkIfPngsMatch(
           sessionReportPath,
           getBaselineReportPath(reportFileName, 'png'),
           config.get('screenshots.directory'),
           log
         );
 
-        expect(percentSimilar).to.be.lessThan(0.1);
+        expect(percentDiff).to.be.lessThan(0.08);
+      });
+    });
+
+    describe('Preserve Layout', () => {
+      it('downloads a PDF file', async function () {
+        // Generating and then comparing reports can take longer than the default 60s timeout because the comparePngs
+        // function is taking about 15 seconds per comparison in jenkins.
+        this.timeout(300000);
+        await PageObjects.common.navigateToApp('dashboard');
+        await PageObjects.dashboard.loadSavedDashboard('Ecom Dashboard');
+        await PageObjects.reporting.openPdfReportingPanel();
+        await PageObjects.reporting.clickGenerateReportButton();
+
+        const url = await PageObjects.reporting.getReportURL(60000);
+        const res = await PageObjects.reporting.getResponse(url);
+
+        expect(res.status).to.equal(200);
```

Comment:
Note

Unfortunately, all that the PDF tests are doing is verifying that the job completes successfully. That is ensured by the 200 response.

@elastic/kibana-app-services we need to think of a good way to validate the contents of a PDF file against a baseline. We have an implementation for PNG but not PDF.